packages/theme/api-report.md

```diff
@@ -41,6 +41,12 @@ export type BackstagePaletteAdditions = {
     indicator: string;
     color: string;
     selectedColor: string;
+    navItem: {
```

Comment:
Unfortunately this is a breaking change that we can't introduce straight away. Do you see some other option here like using something else that already exists in the palette? Perhaps something from the base MUI palette? Otherwise we'd need to make these optional along with a fallback.

Reply:
Sadly I can't seem to find another suitable option. Do you have any suggestions on how we would go about making these optional?

Reply:
Update: I made them into optional props, should resolve this issue